vue/components/ui/molecules/notification/notification.vue

```diff
@@ -32,36 +38,45 @@
     text-align: center;
     top: 100px;
     z-index: 50;
+    transition: opacity 0.125s ease-in-out;

```

Comment:
Do we need both transition and animation?

Reply:
Yes, the animation is for element-enter and the transition (opacity) is for element-leave